alot/completion/__init__.py

```diff
@@ -0,0 +1,17 @@
+from .completer import Completer
```

Comment:
Why all these imports? They look unused to me.

Reply:
yes they are. I just wanted to make sure nothing breaks by making these names available through `alot.completion` as before..